parser/parser.coffee

```diff
@@ -160,11 +161,10 @@ parseVarDec = ->
   new VariableDeclaration id, exp
 
 parseVarAssig = ->
-  left = parseTernExp()
```

Comment:
This doesn't work - for a variable assignment, we need to have an ID or a tuplit on the left side - not any sort of expression.